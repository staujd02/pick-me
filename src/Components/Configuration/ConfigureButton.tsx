import React from 'react';

export default class ConfigureButton extends React.Component<ConfigureButtonProps> {
    render(){
        const { buttonClickHandler } = this.props;
        return <button onClick={buttonClickHandler}>Configure Streams</button>
    }
}