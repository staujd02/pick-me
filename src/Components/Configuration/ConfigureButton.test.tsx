import React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import { formatHTML } from '../../TestUtilities/htmlFormatter';
import ConfigureButton from './ConfigureButton';

describe('The Load Screen', () => {
    
    let wrapper: ShallowWrapper<ConfigureButtonProps,{},ConfigureButton>;

    beforeEach(() => {
        wrapper = shallow(<ConfigureButton buttonClickHandler={jest.fn()} />)
    });

    it('renders correctly', () => expect(formatHTML(wrapper.html())).toMatchSnapshot());
});