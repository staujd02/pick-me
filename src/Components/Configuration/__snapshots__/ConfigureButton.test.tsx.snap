// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Load Screen renders correctly 1`] = `
"
<button>Configure Streams</button>
"
`;
